src/test/java/com/coherent/zipcode/userController/UpdateUser.java

```diff
@@ -0,0 +1,42 @@
+package com.coherent.zipcode.userController;
+
+import com.coherent.user.PatchRequestBody;
+import com.coherent.user.User;
+import com.coherent.zipcode.BasicTestClass;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class UpdateUser extends BasicTestClass {
+
+    @DisplayName("Scenario 1 - User is updated")
+    @Test
+    public void shouldUpdateUser() throws IOException {
+        //TODO for future: try some randomizer class
+        String jsonUser1 = "src/main/resources/payload/createUserAllFields.json";
+        User userToChange = new ObjectMapper().readValue(new File(jsonUser1), User.class);
```

Comment:
You added ObjectMapper to the BasicTestClass. No need to use new ObjectMapper().